src/gwt/tools/i18n-helpers/create_dev_locale.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# Creates a "_dev" locale, which is a copy of the _en locale with PREFIX prepended
+# onto all strings for easy spotting in the UI
+# 
+# Running this script will result in a "XXX_dev.properties" file for each 
+# "XXX_en.properties" file in this directory and its subdirectories
+
+# Note: run this from /src/gwt/src, otherwise you'll create extra properties files
+# from places that are generated code rather than input code
+
+PREFIX="@"
+
+for src in $(find . -name "*en.properties"); do
+  tgt=$(echo "$src" | sed -e 's/_en.properties$/_dev.properties/')
+  echo "Copying $src -> $tgt"
+  if [ -f "$tgt" ]; then
+  	echo "WARNING: Target $tgt already exists - skipping"
+  else
+  	# "copy" src with sed, replacing all lines of "X = Y" with "X = {PREFIX}Y"
+	sed -E "s/^([^\n=]*)=([ ]*)([^\n]*)\$/\1=\2${PREFIX}\3/" $src > $tgt
+  fi
+done
```

Comment:
I ran the script from `src/gwt/src` and it generated `AboutDialogMessages_dev.properties` with `title = @About {0}` as expected, but the resulting `AboutDialogConstants_dev.properties` didn't have the leading `@`.

Maybe another macOS quirk with `sed`? Here's example from the terminal:

```
➜  ui git:(locale/barebones-intl) ✗ sed -E "s/^([^\n=]*)=([ ]*)([^\n]*)\$/\1=\2@\3/" AboutDialogMessages_en.properties
title = @About {0}
➜  ui git:(locale/barebones-intl) ✗ sed -E "s/^([^\n=]*)=([ ]*)([^\n]*)\$/\1=\2@\3/" AboutDialogConstants_en.properties 
okBtn = Ok
manageLicenseBtn = Manage License
```

If you don't spot something obvious (and/or don't have a Mac to try it on) let me know and I'll look closer.

Reply:
Sorry didn't even think of that.  I don't have a Mac to try it on, but I think the `sed` on Mac has a few differences.  Internet tells me things like `*` or `\n` might behave differently, but without a Mac I'm just guessing.  From what I read, I'm shocked it worked at all...

Reply:
This worked for me (might also work on Linux but I didn't test that so kept separate). Used `\x0D` instead of `\n` and had to drop the `\$` or it wouldn't match a line with no newline (e.g. AboutDialogMessages_en.properties has no newline at the end of file).

```bash
    # "copy" src with sed, replacing all lines of "X = Y" with "X = {PREFIX}Y"
    if [[ "$OSTYPE" = "darwin"* ]]; then
      sed -E "s/^([^\x0D=]*)=([ ]*)([^\x0D]*)/\1=\2${PREFIX}\3/" $src > $tgt
    else
      sed -E "s/^([^\n=]*)=([ ]*)([^\n]*)\$/\1=\2${PREFIX}\3/" $src > $tgt
    fi
```